tensorboard/plugins/image/images_plugin_test.py

```diff
@@ -47,6 +45,15 @@
 
 
 class ImagesPluginTest(tf.test.TestCase):
+    def setUp(self):
+        super().setUp()
+        (logdir, multiplexer) = self._create_data()
+        provider = data_provider.MultiplexerDataProvider(multiplexer, logdir)
+        ctx = base_plugin.TBContext(logdir=logdir, data_provider=provider)
+        plugin = images_plugin.ImagesPlugin(ctx)
+        self._initialize_plugin_specific_attrs(plugin)
```

Comment:
This `setUp()` would probably read better with this call inlined now that it's only used once, but up to you since it's existing code

Reply:
I was considering it, and am happy to do so. More red.